### PR TITLE
[SMALLFIX] Add integration tests for ACL sync with object storages

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

**Comment by ifcharming** [comment]
@calvinjia

File: tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java [hunk 1]
```diff
@@ -388,4 +392,35 @@ public void parentDirectoryPermissionForUfsTest() throws IOException {
...
+    Assert.assertEquals("", sUfs.getOwner(PathUtils.concatPath(sUfsRoot, fileA)));
+    Assert.assertEquals("", sUfs.getGroup(PathUtils.concatPath(sUfsRoot, fileA)));
+    Assert.assertEquals(Constants.DEFAULT_FILE_SYSTEM_MODE,
+        sUfs.getMode(PathUtils.concatPath(sUfsRoot, fileA)));
+
+    // chown and chmod to Alluxio file would not affect the permission of UFS file.
+    sTFS.setOwner(fileA, newOwner, newGroup);
+    sTFS.setPermission(fileA, FsPermission.createImmutable((short) 0700));
+    Assert.assertEquals(Constants.DEFAULT_FILE_SYSTEM_MODE,
+        sUfs.getMode(PathUtils.concatPath(sUfsRoot, fileA)));
```
Comment (calvinjia) [inline tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java]: Do we also want to check the owner/group is unchanged?

Comment (ifcharming) [inline tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java]: good catch, updated. thanks.

**Comment by ifcharming** [comment]
Thanks @calvinjia for the review. I've updated the PR. Can you PTAL?

File: tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java [hunk 2]
```diff
@@ -388,4 +392,37 @@ public void parentDirectoryPermissionForUfsTest() throws IOException {
...
         (int) sUfs.getMode(PathUtils.concatPath(sUfsRoot, fileB.getParent())));
   }
+
+  /**
+   * Tests for the permission of object storage UFS files.
+   */
+  @Test
+  public void objectStoragePermissionTest() throws Exception {
+    if (!(sUfs instanceof S3UnderFileSystem) && !(sUfs instanceof S3AUnderFileSystem)
+        && !(sUfs instanceof SwiftUnderFileSystem) && !(sUfs instanceof GCSUnderFileSystem)) {
```
Comment (calvinjia) [inline tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java]: Should OSS be in this list?

**Comment by calvinjia** [comment]
@ifcharming One minor comment otherwise looks good.

File: tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java (hunk 2 above)
Comment (ifcharming) [inline tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java]: done

**Comment by ifcharming** [comment]
@calvinjia  added OSS, thanks! PTAL

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**